res/css/views/rooms/_DecryptionFailureBar.pcss

```diff
@@ -15,46 +15,70 @@ limitations under the License.
 */
 
 .mx_DecryptionFailureBar {
+    --gap: $spacing-8 $spacing-12;
+
+    display: grid;
+    grid-template-areas:
+        "indicator header  header"
+        ".         message message"; /* Empty grid item under grid-area "indicator" */
```

Comment:
Please note that leaving empty space under indicator would not be possible with flexbox.